common/onepanel/base/cluster-role.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Can the same `ClusterRole` as `argo` sa be used here? Also can we just rename `argo` sa to `onepanel` if we can indeed use the same role?

Reply:
Will test

Reply:
ClusterRole from argo was used as reference, so this comment is completed.

Reply:
@aleksandrmelnikov I actually meant use the exact same `clusterrole` and use `onepanel` as the user for argo.

Reply:
disregard, let's keep them separate for now.

Reply:
Ahhh okay